src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml

```diff
@@ -70,43 +72,34 @@
             <ToggleSwitch x:Name="Toggle_PowerRename_RestoreFlagsOnLaunch"
                           Header="Restore search, replace and flags values on launch from previous run"     
                           Margin="{StaticResource SmallTopMargin}" 
-                          Toggled="Toggle_PowerRename_RestoreFlagsOnLaunch_Toggled"
+                          IsOn="{Binding Mode=TwoWay, Path=RestoreFlagsOnLaunch}"
                            />
 
             <muxc:NumberBox x:Name="Toggle_PowerRename_MaxDispListNum"
                             Header="Maximum numbers of items to show in recently used list"
                             SpinButtonPlacementMode="Inline"
                             HorizontalAlignment="Left"
                             Margin="{StaticResource SmallTopMargin}" 
-                           ValueChanged="Toggle_PowerRename_MaxDispListNum_ValueChanged"
+                            Value="{Binding Mode=TwoWay, Path=MaxDispListNum}"
                             />
         </StackPanel>
-        
-        
-        <StackPanel 
-            x:Name="SidePanel"
-            Orientation="Vertical"
-            HorizontalAlignment="Left"
-            Width="{StaticResource SidePanelWidth}"
-            Grid.Column="1">
 
-            <TextBlock 
-                Text="About this feature"
-                Style="{StaticResource SettingsGroupTitleStyle}"
-                Margin="{StaticResource XSmallBottomMargin}"/>
 
-            <HyperlinkButton 
-                Content="Module overview" 
-                NavigateUri="https://github.com/microsoft/PowerToys/tree/master/src/modules/powerrename"/>
+        <StackPanel x:Name="SidePanel"
+                    Orientation="Vertical"
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
+
+            <TextBlock Text="About this feature"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
+                       Margin="{StaticResource XSmallBottomMargin}"/>
 
-            <TextBlock 
-                Text="Attribution"
-                Style="{StaticResource SettingsGroupTitleStyle}" />
+            <HyperlinkButton Content="Module overview"
+                             NavigateUri="https://github.com/microsoft/PowerToys/tree/master/src/modules/powerrename"/>
 
-            <HyperlinkButton 
```

Comment:
I thin you accidently removed this.

Reply:
https://github.com/microsoft/PowerToys/pull/2129/commits/dd4b43c2088b756dad22be3831f8112a7db709da